src/components/users/form/UserUpdateForm.tsx

```diff
@@ -0,0 +1,135 @@
+import { useForm } from 'react-hook-form';
+import { Button, Form, ModalContent, Fieldset, FormActions, FormAction, FormInput } from '@taskany/bricks';
+import { zodResolver } from '@hookform/resolvers/zod';
+import styled from 'styled-components';
+
+import { User } from '../../../api-client/users/user-types';
+import { userUpdateSchema } from '../../schema/user';
+import { UpdateUserDto } from '../../../api-client/users/update-user-dto';
+import { notifyPromise } from '../../../utils/notifyPromise';
+import { trpc } from '../../../trpc/trpcClient';
+
+import { tr } from './form.i18n';
+
+type UserDataFormProps = {
+    user: User;
+    onClose: () => void;
+};
+
+const NoWrap = styled.div`
+    white-space: nowrap;
+`;
+
+const telegramPrefixes = ['@', 'https://t.me/'];
+
+const UserUpdateForm = ({ user, onClose }: UserDataFormProps) => {
+    const userForm = useForm<UpdateUserDto>({
+        defaultValues: {
+            fullName: user.fullName,
+            email: user.email,
+            supervisorId: user.supervisor?.userId,
+            github: user.github,
+            telegram: user.telegram,
+        },
+        resolver: zodResolver(userUpdateSchema),
+    });
+    const userUpdate = trpc.user.update.useMutation();
```

Comment:
maybe better to move it and updateUser to user-api-hook?

Reply:
No, we'll get rid of that file completely as it currently used only on the back-end